comp/otelcol/otlp/components/exporter/serializerexporter/exporter.go

```diff
@@ -87,7 +88,10 @@ func translatorFromConfig(
 		metrics.WithFallbackSourceProvider(hostGetter),
 		metrics.WithHistogramMode(histogramMode),
 		metrics.WithDeltaTTL(cfg.DeltaTTL),
-		metrics.WithOTelPrefix(),
+	}
+
+	if !pkgdatadog.MetricRemappingDisabledFeatureGate.IsEnabled() {
+		options = append(options, metrics.WithOTelPrefix())
 	}
```

Comment:
This adds a dependency between OSS collector / OTel agent . We can't fill the feature gate to true just in OTel Agent. Can we avoid that ?

Reply:
We can flip it to true in otel agent, we did the same for operation name v2, see https://github.com/DataDog/datadog-agent/blob/e305652b1fe275fb27009d4e6153cacecb503ae4/cmd/otel-agent/command/command.go#L110